tests/unit/test_network_connections.py

```diff
@@ -3670,6 +3707,95 @@ def test_path_to_glib_bytes(self):
         self.assertEqual(result.get_data(), b"file:///my/test/path\x00")
 
 
+class TestValidatorMatch(unittest.TestCase):
+    def setUp(self):
+        self.test_profile = {
+            "name": "test",
+            "type": "ethernet",
+            "ip": {
+                "dhcp4": False,
+                "address": "192.168.245.7/24",
+            },
+            "match": None,
+        }
+        self.validator = network_lsr.argument_validator.ArgValidator_DictConnection()
+
+    def test_match_path_empty_list(self):
+        """
+        Test that 'match.path' setting can be defined as None, [], [""], [None] or ""
+        and such a setting will be normalized into []
+        """
+        for disabled_path in [None, [], [""], [None], ""]:
+            self.test_profile["match"] = {"path": disabled_path}
+            result = self.validator.validate(self.test_profile)
+            self.assertEqual(result["match"], {"path": []})
+
+    def test_match_path_valid_setting(self):
```

Comment:
Please also add a test for

```yaml
match:
  path:
    - ""
    - usb123
    - None
```

I guess this should be normalized into `match: path: ["usb123"]`.

Reply:
Yes. Test added. 
```yaml
    def test_match_path_setting_normalization(self):
        """
        Test that 'match.path' setting ["", "usb123", None] will be normalized into
        ["usb123"]
        """
        self.test_profile["match"] = {"path": ["", "usb123", None]}
        result = self.validator.validate(self.test_profile)
        self.assertEqual(result["match"], {"path": ["usb123"]})
```